client/my-sites/plans/jetpack-plans/product-store/item-list/index.tsx

```diff
@@ -11,14 +14,27 @@ export const ItemsList: React.FC< ItemsListProps > = ( {
 	onClickPurchase,
 	siteId,
 } ) => {
+	const { currentItem, setCurrentItem, clickMoreHandlerFactory } = useItemLightbox();
+
 	return (
 		<div className="jetpack-product-store__items-list">
+			{ isEnabled( 'jetpack/pricing-page-product-lightbox' ) && currentItem && (
```

Comment:
This lightbox feature-flag check should be removed too.

Reply:
> This is great @jkguidaven, The lightboxes work great on the new pricing page!
> Although I think there might be some confusion about the lightbox feature-flag. The lightbox feature-flag is/was strictly for displaying the lightbox(s) on the current (_old_) pricing page. The new pricingPage (productStore), should display the lightbox without needing/using a separate lightbox feature-flag. See comments below.

thank you for clarifying this @elliottprogrammer. I have completely removed the feature flag in this PR.